mne_qt_browser/_pg_figure.py

```diff
@@ -1809,19 +1809,26 @@ def __init__(self, main, title="Settings", **kwargs):
         ch_scaling_layout = QFormLayout()
         self.ch_scaling_spinboxes = {}
 
-        # Get all unique channel types
+        # Get all unique channel types and allow scaling
+        scaler = 1 if self.mne.butterfly else 2
         ordered_types = self.mne.ch_types[self.mne.ch_order]
         unique_type_idxs = np.unique(ordered_types, return_index=True)[1]
         ch_types_ordered = [ordered_types[idx] for idx in sorted(unique_type_idxs)]
         for ch in ch_types_ordered:
             if ch in self.mne.unit_scalings.keys():
                 ch_spinbox = QDoubleSpinBox()
+                ch_spinbox.setMinimumWidth(100)
                 ch_spinbox.setRange(-float("inf"), float("inf"))
-                ch_spinbox.setDecimals(3)
-                ch_spinbox.setValue(self.mne.scalings[ch] * self.mne.unit_scalings[ch])
-                ch_spinbox.setDisabled(True)
-                # ch_spinbox.setReadOnly(True)
-                ch_spinbox.setMinimumWidth(150)
+                ch_spinbox.setDecimals(1)
+                ch_spinbox.setValue(
+                    self.mne.scalings[ch]
+                    * self.mne.unit_scalings[ch]
+                    * scaler
+                    / self.mne.scale_factor
```

Comment:
... and this are identical (as are the `scaler` computations). Better to refactor into a helper I think as I commented on one of your commits